server/eventlog.go

```diff
@@ -6,6 +6,7 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/google/certificate-transparency-go/x509"
```

Comment:
short answer: go-attestation depends on it. long answer: some EK vendor roots are malformed and `crypto/x509` doesn't parse them. In particular, the CT package comes with its own Cert struct. Unfortunately, go-attestation also decided to use it for the SecureBoot parsing functionality rather than only use the CT x509 in EK verification.

Reply:
Thanks!